### PR TITLE
Add #asMethodReturningString:named:

### LINKED ISSUE #141: #asMethodReturningByteArray:named: needs String equivalent
We need an equivalent of `GRPlatform >> #asMethodReturningByteArray:named:` for strings. The reason for this is some platforms like VASt support Unicode but not Unicode string literals.

### PR DESCRIPTION
**Pull request by marschall**
Fixes #141

### CODE REVIEW

File: repository/Grease-Core.package/GRPlatform.class/instance/asMethodReturningString.named..st [hunk 1]
```diff
@@ -0,0 +1,21 @@
+file library
+asMethodReturningString: aByteArrayOrString named: aSymbol
```
Comment (marschall) [inline repository/Grease-Core.package/GRPlatform.class/instance/asMethodReturningString.named..st:2]: I decided to add a default implementation that works for anybody that does not need special behaviour.

Comment (jbrichau) [reply repository/Grease-Core.package/GRPlatform.class/instance/asMethodReturningString.named..st:2]: Could we also include a test for this? It will also verify the correct behaviour in all tested platforms in the github ci.

Comment (marschall) [reply repository/Grease-Core.package/GRPlatform.class/instance/asMethodReturningString.named..st:2]: > Could we also include a test for this?

I can try. The trick is to figure out whether the dialect supports Unicode.

File: repository/Grease-Tests-Core.package/GRPlatformTest.class/instance/runCase.st [hunk 2]
```diff
@@ -0,0 +1,3 @@
+running
+runCase
+	GRPlatform current doSilently: [ super runCase ]
```
Comment (jbrichau) [inline repository/Grease-Tests-Core.package/GRPlatformTest.class/instance/runCase.st:3]: Hey @marschall 
Question: why is this needed and how does it work? the `doSilently:` does nothing but evaluation the block?

Comment (marschall) [reply repository/Grease-Tests-Core.package/GRPlatformTest.class/instance/runCase.st:3]: > Hey @marschall
> Question: why is this needed and how does it work? the `doSilently:` does nothing but evaluation the block?

That's only the default implementation. On Pharo it suspends the announcements preventing the package from becoming dirty

https://github.com/SeasideSt/Grease/blob/master/repository/Grease-Pharo100-Core.package/GRPharoPlatform.class/instance/doSilently..st

This is used by `WAFileLibraryTest` where the `ByteArray` equivalent is tested

https://github.com/SeasideSt/Seaside/blob/master/repository/Seaside-Tests-Pharo-Core.package/WAFileLibraryTest.extension/instance/runCase.st

Comment (jbrichau) [reply repository/Grease-Tests-Core.package/GRPlatformTest.class/instance/runCase.st:3]: Sorry, I should have read this in an image and I would have realised this is not a new method. Now I was just reading the changes on github and assumed this was newly introduced.

**Review by jbrichau — APPROVED** [review]
